web/components/issues/issue-detail/issue-activity/comments/comment-create.tsx

```diff
@@ -82,7 +82,7 @@ export const IssueCommentCreate: FC<TIssueCommentCreate> = (props) => {
             render={({ field: { value, onChange } }) => (
               <LiteTextEditorWithRef
                 onEnterKeyPress={(e) => {
-                  handleSubmit(onSubmit)(e);
+                  !(watch("comment_html") === "") && handleSubmit(onSubmit)(e);
```

Comment:
@prateekshourya29 , could you please address all edge cases? For instance, if a user enters only blank spaces or if there are spaces at the end of their comment, could you trim those as well? It seems that your current approach may not function as intended.

Reply:
Sure @anmolsinghbhatia